### ISSUE TITLE
Add RDNA2 to  gpu buyers guide

### ISSUE THREAD

**Issue by TheHackGuy** [dortania/bugtracker#231]
The RDNA2 cards from the rx6000 lineup are listed unsupported while RDNA2 is now supported since big sur 11.4

Gpus that are supported:  6800, 6800XT and 
         6900XT

https://dortania.github.io/GPU-Buyers-Guide/modern-gpus/amd-gpu.html#unsupported-amd-gpus

**Comment by ArchCryptonIO** [comment]
You may check any kext related  to this line up....and find device-id related your GPU. If the device-id not exist....im sure the card is not support yet.  The way to find future support dGPU is by using hex editor and open executable file inside kext related. Sometimes Apple put some hint for future support but not listed on info.plist.

**Comment by TheHackGuy** [comment]
yes I know. I just want that they add the gpu's to the support list for other people.
